embassy-stm32/src/usb/otg.rs

```diff
@@ -1,5 +1,6 @@
 use core::marker::PhantomData;
 
+use crate::pac::rcc::vals::Otghssel;
```

Comment:
Given that I added this value to the PAC create I think it was behind a feature flag as not all chips actually contain this clock. Looks like CI is also complaining about it on unrelated (non STM32U5) boards.

Reply:
Removed. Not needed now.